### ISSUE TITLE
Download Issue

### ISSUE THREAD

**Issue by Dedwing** [cloudtostreet/Sen1Floods11#11]
I downloaded the Sen1Flood11 dataset and managed to get all subdirectories. However, when I go to open some of the flood_events data, the majority of it is black and white. I'm new to this, so is this what I am supposed to be seeing? I thought I would see the original Sentinel Imagery from which the flooding pixels were derived from. Could someone just explain to me what I should be seeing? Thank you!

**Comment by Haomiaoyanmeng** [comment]
Yes, I have the same question. What i've seen is black and white, i wondering if this is right? and have you ever solved this issue??

**Comment by Haomiaoyanmeng** [comment]
OOO, i've seen the s1 and s2 data, all of these images are very dark, we need to normalize them to better see them.
